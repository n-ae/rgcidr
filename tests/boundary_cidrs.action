0.0.0.0/0 tests/boundary_ips.given
