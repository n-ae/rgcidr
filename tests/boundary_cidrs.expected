0.0.0.0
1.2.3.4
127.0.0.1
192.168.1.1
255.255.255.255
